lox_services/persistence/database/remove_duplicates.py

```diff
@@ -114,6 +115,70 @@ def remove_duplicate_refunds(dataframe: pd.DataFrame) -> pd.DataFrame:
     return dataframe
 
 
+def remove_duplicate_deliveries(dataframe: pd.DataFrame) -> pd.DataFrame:
+    """
+    Removes duplicate rows from the provided DataFrame based on certain criteria.
+
+    This function checks for duplicate deliveries in the input DataFrame by comparing
+    the combination of tracking number, status, and formatted datetime. If duplicates
+    are found, they are removed from the DataFrame. The function also ensures that the
+    datetime column is formatted correctly for comparison.
+
+    Args:
+        dataframe (pd.DataFrame): The input DataFrame containing delivery data.
+
+    Returns:
+        pd.DataFrame: A DataFrame with duplicate deliveries removed, if any.
+    """
+    print("Checks for duplicate rows in db..")
```

Comment:
NIT: I would put print after empty check.

Reply:
There is already one print in the insert function

Reply:
Ok I got your point now